### ISSUE TITLE
Intervene on Raid Frames

### ISSUE THREAD

**Issue by Humdibum** [jordonwow/bigdebuffs#241]
Intervene on Raid Frames would be very nice for Arena

**Comment by MIKBT** [comment]
I agree. Intervene and Spell reflect does not show in Friendly Unit Frames. I have set both to show but it does not work. Please fix it, I want to be able to see when I intervene someone in arena and know how much left they have and the same with spell reflect of overwatch and the legendary.

**Comment by domzae** [comment]
Hey @Humdibum & @MIKBT, unfortunately this is a limitation in the way raid frames are currently handled in this addon - only modifying the way debuffs are shown. (So spell reflect/intervene/any other buff won't show unless it is shown by the default raid frames anyway). Changing the way the addon works to also show/modify buffs on raid frames is a lot of work, so it might not happen anytime soon, if ever.

The good news: there are relatively simple solutions in the meantime! The first, is to enable Unit Frames > Party Frames, and use a manual anchor. You can use this in addition to enabled Raid Frames, but means you would get duplicates of the top priority debuff: 
![image](https://user-images.githubusercontent.com/40212333/111868009-d99d2480-8977-11eb-8ed2-f0840517ded7.png)

Another option is to use an addon that allows you to show specific auras on raid frames. For example, I use [Enhanced Raid Frames](https://www.curseforge.com/wow/addons/enhanced-raid-frames), which doesn't change the look of the raid frames, but allows you to add indicators: 
![image](https://user-images.githubusercontent.com/40212333/111868290-65fc1700-8979-11eb-91d1-62c50f802eb9.png)
(It's pretty straight forward, but if you want a hand setting this up feel free to send me a message on discord Domzae#8909)

**Comment by MIKBT** [comment]
Enhanced Raid Frames does the job for the moment. Many thanks your help hero!
